libs/common/src/assets/i18n/en.json

```diff
@@ -263,12 +263,21 @@
 	"ontology.delete_warning": "Deleting this label set will make you lose all existing classifications",
 	"ontology.delete_warning_extra": "If you delete this classification it will disappear from all resources and search suggestions",
 	"ontology.duplicated_label": "A label named <strong>{{title}}</strong> already exists",
+	"ontology.excluding_labels": "Excluding labels",
 	"ontology.invalid_name": "Invalid label name",
 	"ontology.label_classification": "Label classification",
 	"ontology.manage_labels": "Add, name, and sort the labels that are part of this Label Set.",
 	"ontology.ontology_name": "Label set name",
+	"ontology.paragraphs": "Paragraphs",
+	"ontology.resources": "Resources",
+	"ontology.sentences": "Sentences",
 	"ontology.sort_alphabetically": "Sort Alphabetically",
 	"ontology.tags_to_label_button": "Create tags from folders",
+	"ontology.type_classification": "Type of classification",
+	"ontology.type_classification_important": "IMP! Irreversible option. It cannot be modified later.",
```

Comment:
`IMP!` feels super weird, like why a contraction? also, "warning" is more commonly used than "important" I think.

Reply:
Fixed 👍